src/runtime_src/core/edge/drm/zocl/zocl_kds.c

```diff
@@ -590,6 +590,10 @@ static void notify_execbuf(struct kds_command *xcmd, int status)
 		ecmd->state = ERT_CMD_STATE_TIMEOUT;
 	else if (status == KDS_ABORT)
 		ecmd->state = ERT_CMD_STATE_ABORT;
+	else if (status == KDS_SKERROR)
+		ecmd->state = ERT_CMD_STATE_SKERROR;
+	else if (status == KDS_SKCRASHED)
+		ecmd->state = ERT_CMD_STATE_SKCRASHED;
 
```

Comment:
It is best to convert this into a table look up or at the minimum a separate function which translates from KDS state to ERT state.